README.md

```diff
@@ -166,7 +166,7 @@ List of currently supported languages:
 - [x] html (maintained by @TravonteD)
 - [ ] csharp
 - [ ] swift
-- [ ] java
+- [x] java
```

Comment:
Please add your name here if you want to maintain this language.

Reply:
Maybe later? After August?